src/estimate/estimator.py

```diff
@@ -61,14 +61,14 @@ def handle_request(data):
     if output_type.name not in loaded_model:
         loaded_model[output_type.name] = dict()
     output_path = ""
-    request_trainer = True
+    request_trainer = False
     if power_request.trainer_name is not None:
         if output_type.name in loaded_model and power_request.energy_source in loaded_model[output_type.name]:
             current_trainer = loaded_model[output_type.name][power_request.energy_source].trainer_name
-            request_trainer = current_trainer == power_request.trainer_name
-            if not request_trainer:
+            request_trainer = not current_trainer == power_request.trainer_name
```

Comment:
Did you mean 
```suggestion
            request_trainer = current_trainer != power_request.trainer_name
```

Reply:
Yes, I confused the syntax a little bit. Changed to `!=`.